src/index.css

```diff
@@ -1,11 +1,66 @@
-body {
-  font-family: sans-serif;
-  margin: 0;
-}
-
 html,
 body,
 #root {
   height: 100%;
-  width: 100%;
+  width: 100vw;
+  display:grid;
+  grid-template-columns: 34% repeat(2, 33%);
+  grid-template-rows: 50px auto auto;
+  grid-template-areas: "ga1 ga1 ga1" "ga2 ga2 ga3" "ga4 ga4 ga3";
```

Comment:
These are not the best names for our grid areas;

Also - this is _stikingly similar_ to another groups CSS rules.

We have no problem with ya'll working together but make sure to site from whom or where you got this code.